features/YellowBox.feature

```diff
@@ -6,77 +6,70 @@ Feature: Managing Features with the YellowBox
 
   Background:
     Given the store operates on a single channel in "EUR" currency
+    And I am on the yellow-box page
+    And there is no yellowbox cookie
 
   @ui
-  Scenario: Visit the shop-page
-    When I am on the yellow-box page
+  Scenario: I Visit the shop-page
     Then I should see the yellow-box overlay
 
   @ui @javascript
-  Scenario: open and close the overlay
-    When I am on the yellow-box page
+  Scenario: I open and close the overlay
     Then I should see the yellow-box overlay
-    And i click on the yellow-box symbol
-    Then i should see the expanded yellow-box overlay
-    And i click on the close button
-    Then i should not see the expanded yellow-box overlay
+    And I click on the yellow-box symbol
+    Then I should see the expanded yellow-box overlay
+    And I click on the close button
+    Then I should not see the expanded yellow-box overlay
```

Comment:
Warum denn schon wieder Then und nicht einfach nur And?

Reply:
Es sieht ein bisschen so aus, als ob die Ands nach den Thens wieder Aktionen sind die wieder überprüft werden mit dem nächsten Then. Was in meinen Augen allerdings eher ein Zeichen dafür ist, dass hier mehrere Szenarien daraus gemacht werden sollten.

Z.B. eines für das Öffnen des Overlays und eines für das Schliessen.